From martyn.welch@gefanuc.com  Fri Dec 18 10:24:37 2009
From: Martyn Welch <martyn.welch@gefanuc.com>
Date: Tue, 15 Dec 2009 08:43:09 +0000
Subject: Staging: vme: Remove old structures for implemented features
To: greg@kroah.com
Cc: devel@linuxdriverproject.org
Message-ID: <20091215084309.3930.38954.stgit@ES-J7S4D2J.amer.consind.ge.com>


Functionality for master RMW and location monitors has been implemented.
Remove the commented out structures from the original codebase.

Signed-off-by: Martyn Welch <martyn.welch@gefanuc.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/staging/vme/vme_bridge.h |   32 --------------------------------
 1 file changed, 32 deletions(-)

--- a/drivers/staging/vme/vme_bridge.h
+++ b/drivers/staging/vme/vme_bridge.h
@@ -229,36 +229,4 @@ struct vmeArbiterCfg {
 };
 typedef struct vmeArbiterCfg vmeArbiterCfg_t;
 
-
-/*
- *  VMEbus RMW Configuration Data
- */
-struct vmeRmwCfg {
-	unsigned int targetAddrU;	/*  VME Address (Upper) to trigger RMW cycle */
-	unsigned int targetAddr;	/*  VME Address (Lower) to trigger RMW cycle */
-	vme_address_t addrSpace;	/*  VME Address Space */
-	int enableMask;		/*  Bit mask defining the bits of interest */
-	int compareData;	/*  Data to be compared with the data read */
-	int swapData;		/*  Data written to the VMEbus on success */
-	int maxAttempts;	/*  Maximum times to try */
-	int numAttempts;	/*  Number of attempts before success */
-	int reserved;		/* For future use */
-
-};
-typedef struct vmeRmwCfg vmeRmwCfg_t;
-
-/*
- *  VMEbus Location Monitor Arg Structure
- */
-struct vmeLmCfg {
-	unsigned int addrU;	/*  Location Monitor Address upper */
-	unsigned int addr;	/*  Location Monitor Address lower */
-	vme_address_t addrSpace;	/*  Address Space */
-	int userAccessType;	/*  User/Supervisor Access Type */
-	int dataAccessType;	/*  Data/Program Access Type */
-	int lmWait;		/* Time to wait for access */
-	int lmEvents;		/* Lm event mask */
-	int reserved;		/* For future use */
-};
-typedef struct vmeLmCfg vmeLmCfg_t;
 #endif
